localnet/run.ts

```diff
@@ -355,10 +322,48 @@ const main = async () => {
   const [richDudePubKey, richDudeKeypair] = await amman.loadOrGenKeypair(
     'rich-dude',
   );
+  const [tokenCreatorPubkey, tokenCreatorKeypair] =
+    await amman.loadOrGenKeypair('token-creator');
+  const [tokenMintPubkey, tokenMintKeypair] = await amman.genLabeledKeypair(
+    'my-token-mint-keypair',
+  );
+  const [tokenMintAuthPubkey, tokenMintAuthKeypair] =
+    await amman.genLabeledKeypair('my-token-authority-keypair');
+
   await ensureBalance(amman, connection, richDudePubKey, 1000);
+  await ensureBalance(amman, connection, tokenCreatorPubkey, 1000);
+
+  const [tokenMintAddress, tokenAccountAddress] = await createToken(
+    amman,
+    connection,
+    tokenMintKeypair,
+    tokenMintAuthKeypair,
+    tokenCreatorKeypair,
+  );
+
+  await mintTokens(
+    connection,
+    tokenMintPubkey,
+    tokenMintAuthKeypair,
+    richDudeKeypair,
+    richDudePubKey,
```

Comment:
not the most appropriate name to have sitting in your code ^^

Reply:
changed to richPerson, haha